templates/energyplus/templates/ref-sys.pxt

```diff
@@ -41,7 +42,7 @@ Refrigeration:System,
  <%   raise("error: unknown option condenser_type = '#{condenser_type}'") %>
  <% end %>
     <%= ref_sys_comp_list %>,   !- Compressor or CompressorList Name
-    25.0,                    !- Minimum Condensing Temperature {C}
+    <%= cond_min_temp %>,                    !- Minimum Condensing Temperature {C}
```

Comment:
By default (unless `refsys_cond_floating` = true), use the parameter value for minimum condensing temperature.